x-pack/plugins/security_solution/public/users/pages/index.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { Route, Switch, Redirect } from 'react-router-dom';
+import { USERS_PATH } from '../../../common/constants';
+import { UsersTableType } from '../store/model';
+import { Users } from './users';
+import { UsersDetails } from './details';
+
+const usersTabPath = `${USERS_PATH}/:tabName(${UsersTableType.allUsers})`;
+
+import { usersDetailsPagePath } from './types';
+
+const usersDetailsTabPath = `${usersDetailsPagePath}/:tabName(${UsersTableType.allUsers})`;
```

Comment:
is this `user/${user}/allUsers` url temporary?

Reply:
Yes, it doesn't make sense to show all users on a specific user page. It is there while I don't know what to show.